### PR TITLE
D.add Still Returns An Error

### PR DESCRIPTION
**Pull request by svozza**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "date-fp",
-  "version": "3.0.5",
+  "version": "3.0.6",
```
Comment (davidchambers) [inline package.json]: Should this be a major rather than a patch release?

Comment (svozza) [inline package.json]: No, I don't think so. The breaking change was made when we moved from 1.x to 2.x in #22. The fact that this function was overlooked is a bug.

Comment (davidchambers) [inline package.json]: Yeah, I can see it both ways. It's certainly unfortunate that someone depending on `date-fp@3.x.x` and relying on the 2.0.0–3.0.5 behaviour will have her code broken by this change.

Comment (svozza) [inline package.json]: That's a fair point. I don't really have a problem jumping to `4.0`. @cullophid What do you think?

Comment (cullophid) [inline package.json]: When in doubt Its a major :)

Comment (svozza) [inline package.json]: :zap:

**Pull request merged**